test/support/index_document_test_helpers.rb

```diff
@@ -0,0 +1,168 @@
+module IndexDocumentsTestHelpers
+  # There are quite a few api calls made when expanding/indexing
+  # data when topics are updated.
+
+  # Hide the stub requests in this helper
+  def stub_calls_for_index_documents_test
+    stub_request(:get, "http://localhost:9200/mainstream_test/edition/%2Ftags%2Fspecialist_sector%2Fanimal-welfare%2Fpets").
+      to_return(:status => 200, :body => pets_document)
+
+    stub_request(:get, "http://localhost:9200/mainstream_test/edition/%2Ftopic%2Fanimal-welfare%2Fpets").
+      to_return(:status => 200, :body => pets_document)
+
+    stub_request(:get, "http://localhost:9200/page-traffic/_aliases").
+      to_return(:status => 200, :body => page_traffic)
+
+    stub_request(:get, "http://localhost:9200/page-traffic/_search").
+      with(:body => page_traffic_search).
+      to_return(:status => 200, :body => page_traffic_search_results)
+
+    stub_request(:get, "http://publishing-api.dev.gov.uk/v2/content/cc9eb8ab-7701-43a7-a66d-bdc5046224c0").
+      to_return(:status => 200, :body => cc9eb_response)
+
+    stub_request(:get, "http://localhost:9200/page-traffic/_search").
+      with(:body => "{\"query\":{\"match_all\":{}},\"size\":0}").
+      to_return(:status => 200, :body => page_traffic_search_yet_another_call)
+
+    stub_request(:post, "http://localhost:9200/mainstream_test/_bulk").
+      with(:body => post_to_elasticsearch).
+      to_return(:status => 200, :body => elasticsearch_post_response)
+
+    stub_request(:post, "http://localhost:9200/mainstream_test/_bulk").
+      with(:body => post_no_topics_to_elasticsearch).
+      to_return(:status => 200, :body => no_topics_elasticsearch_post_response)
+  end
+
+  def post_to_elasticsearch
+    "{\"index\":{\"_type\":\"edition\",\"_id\":\"/topic/animal-welfare/pets\"}}\n{\"popularity\":5.6085249579360626e-05,\"slug\":\"animal-welfare/pets\",\"topics\":[\"/ooo/aar\"],\"description\":\"Info about pets.\",\"format\":\"specialist_sector\",\"link\":\"/topic/animal-welfare/pets\",\"title\":\"Pets\",\"_type\":\"edition\",\"_id\":\"/topic/animal-welfare/pets\"}\n"
```

Comment:
It's quite hard to read this; perhaps it would be clearer to write this as a nested hash datastructure and call .to_json on it to produce the sample body.

The topics should be being sent to elasticsearch in the `specialist_sectors` field, not the `topics` field.  (The `topics` field contains the things which whitehall calls topics internally, but which are now called "policy areas" in the UI.)